src/content/pt/2019/accessibility.md

```diff
@@ -0,0 +1,317 @@
+---
+part_number: II
+chapter_number: 9
+title: Acessibilidade
+description: Capítulo de acessibilidade do Almanaque da Web de 2019 cobrindo facilidade de leitura, mídia, facilidade de navegação e compatibilidade com tecnologias assistivas.
+authors: [nektarios-paisios, obto, kleinab]
+reviewers: [ljme]
+translators: [edu_qg]
+discuss: 1764
+results: https://docs.google.com/spreadsheets/d/16JGy-ehf4taU0w4ABiKjsHGEXNDXxOlb__idY8ifUtQ/
+queries: 09_Accessibility
+published: 2019-11-11T00:00:00.000Z
+last_updated: 2020-09-21T00:00:00.000Z
+---
+
+## Introdução
+
+A acessibilidade na web é essencial para uma sociedade inclusiva e igualitária. À medida que mais de nossas vidas sociais e profissionais mudam para o mundo online, torna-se ainda mais importante para as pessoas com deficiência poderem participar de todas as interações online sem barreiras. Assim como os arquitetos de edifícios podem criar ou não recursos de acessibilidade, como rampas para cadeiras de rodas, os desenvolvedores da web podem ajudar ou atrapalhar em tecnologias assistivas que os usuários confiam.
```

Comment:
@HakaCode had the folllowing in the `featured_chapter.html` file.

I'll leave you two to decide which is the better wording, but we should be consistent as it's supposed to be a quote from this chapter.

```suggestion
A acessibilidade na web é essencial para uma sociedade inclusiva e igualitária. À medida que mais de nossas vidas sociais e profissionais mudam para o mundo online, torna-se ainda mais importante para as pessoas com deficiência poderem participar de todas as interações online sem barreiras. Assim como os arquitetos de edifícios podem criar ou omitir recursos de acessibilidade, como rampas para cadeiras de rodas, os desenvolvedores da web podem ajudar ou atrapalhar a utilização de tecnologias assistivas.
```

Reply:
I liked the suggestion.

Reply:
So what have we decided here? Change the chapter inline with the `src/templates/pt/2019/featured_chapters.html` quote? Or the other way around?

@eduqg can you do one or the other?

Reply:
I have read the English version again and re-translated. @HakaCode  Can you check if it sounds better? It is in my last commit.

Reply:
Hi, @eduqg I read your re-translation but we need to keep the same translation at the featured_chapter. I'm going to let here my suggested translation based on what you have already done (full paragraph). Then you can choose how to adapt:

>A acessibilidade na web é essencial para uma sociedade inclusiva e igualitária. À medida que mais de nossas vidas sociais e profissionais mudam para o mundo online, torna-se ainda mais importante para as pessoas com deficiência poderem participar de todas as interações online sem barreiras. Assim como os arquitetos de edifícios podem criar ou não recursos de acessibilidade, como rampas para cadeiras de rodas, os desenvolvedores da web podem ajudar ou atrapalhar na adoção de tecnologias assistivas de que os usuários dependem.

Na primeira tradução apontada, acho que tanto o "não" como o "omitir" cumprem o papel semântico ali, o segundo trecho foi que achei mais problemático.